tensorboard/webapp/notification_center/_views/notification_center_component.scss

```diff
@@ -13,6 +13,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+.red-dot {
+  background-color: #d93025;
```

Comment:
If possible, use a material color variable; something like `mat-color($mat-red, 700)` 

In a retrospect, you should change `category-icon` as well